dashboard/test/controllers/pd/workshop_user_management_controller_test.rb

```diff
@@ -0,0 +1,58 @@
+require 'test_helper'
+
+class Pd::WorkshopUserManagementControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
+  setup_all do
+    @workshop_admin = create :workshop_admin
+    @teacher = create :teacher
+    @student = create :student
+    @facilitator = create :facilitator
+    @facilitator_with_course = create(:pd_course_facilitator, course: Pd::Workshop::COURSE_CSF).facilitator
+  end
+
+  def self.test_workshop_admin_only(method, action, success_response, params = nil)
+    test_user_gets_response_for action, user: :student, method: method, params: params, response: :forbidden
+    test_user_gets_response_for action, user: -> {@teacher}, method: method, params: params, response: :forbidden
+    test_user_gets_response_for action, user: -> {@workshop_admin}, method: method, params: params, response: success_response
+  end
```

Comment:
nit: the `success_response` param isn't needed. This method is only called once on line 20. `:success` can be used directly here on line 16 and doesn't need to be provided.